src/components/rangeUsePlanPage/conditions/index.js

```diff
@@ -5,21 +5,32 @@ import { CONDITIONS } from '../../../constants/fields'
 import * as strings from '../../../constants/strings'
 import PermissionsField from '../../common/PermissionsField'
 import { TextArea } from 'formik-semantic-ui'
+import { useUser } from '../../../providers/UserProvider'
 
 const Conditions = ({ plan }) => {
-  const { conditions = '' } = plan || {}
+  const user = useUser()
+  const { conditions = '', proposedConditions = '' } = plan || {}
 
   return (
     <div className="rup__conditions">
       <div className="rup__popup-header">
         <div className="rup__content-title">{strings.CONDITIONS}</div>
-        <InfoTip header={strings.CONDITIONS} content={strings.CONDITIONS_TIP} />
       </div>
       <div className="rup__divider" style={{ marginBottom: 10 }} />
       <div className="rup__row">
         <div className="rup__cell-6" style={{ marginBottom: 40 }}>
+          <div className="rup__info-title">
+            <div className="rup__popup-header">
+              Decision Maker
+              <InfoTip
+                header={strings.CONDITIONS}
+                content={strings.CONDITIONS_TIP_DECISION_MAKER}
+                size="tiny"
+              />
+            </div>
+          </div>
           <PermissionsField
-            permission={CONDITIONS}
+            permission={CONDITIONS.DECISION_MAKER_CONDITIONS}
```

Comment:
I think it makes more sense to call this `CONDITIONS.CONDITIONS`. The conditions field is still the actual field where the conditions on the plan are tracked, it just so happens only the DM can edit it.